sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosPagedFluxOptions.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+/**
+ * Specifies paging options for Cosmos Paged Flux implementation.
+ * @see CosmosContinuablePagedFlux
+ */
+public class CosmosPagedFluxOptions {
```

Comment:
I only see this is used in the implementation of CosmosContinuablePagedFlux but not exposed in the public surface area

as it is in `com.azure.cosmos` is it used in the public surface area?

Reply:
Good point, we should have it in implementation folder, I will move it.

Reply:
Done.

Reply:
This is still in the public package.

Reply:
I see it in implementation package. 

https://github.com/Azure/azure-sdk-for-java/pull/7839/files#diff-2028f0a4901c305ffa200afc82a2cf05